moon/cfc_chat_transit/server/init.moon

```diff
@@ -79,39 +78,47 @@ ChatTransit.ReceiveMessage = (ply, text, teamChat) =>
 
     @Logger\debug "Sent message '#{text}' to websocket"
 
-ChatTransit.PlayerConnected = (data) =>
-    steamName = data.name
-    steamId = data.networkid
+ChatTransit.PlayerInitialSpawn = (ply) =>
+    sendMessage = (attempts=1) ->
+        if attempts >= 5
+            return @Logger\error "PlayerSummary didn't exist in time to send an on-spawn message"
```

Comment:
I think logging and still continuing to send the join alert with the default avatar would be better

Reply:
Yeah good call